diff --git a/ckeditor.config.js b/ckeditor.config.js
index 605fcae..c489c4a 100644
--- a/ckeditor.config.js
+++ b/ckeditor.config.js
@@ -4,6 +4,8 @@
  If you don't do this, you may notice that browser is ignoring all your changes.
  */
 CKEDITOR.editorConfig = function(config) {
+  config.entities_greek = false;
+  config.entities_latin = false;
   config.indentClasses = [ 'rteindent1', 'rteindent2', 'rteindent3', 'rteindent4' ];

   // [ Left, Center, Right, Justified ]

